src/pages/api/projects/[id]/issues/nonprofit.ts

```diff
@@ -0,0 +1,24 @@
+import { Types } from "mongoose";
+import router from "next/router";
+import { nonprofitCreateIssue } from "server/mongodb/actions/Issue";
+import APIWrapper from "server/utils/APIWrapper";
+import { NonprofitCreateIssue, Role } from "src/utils/types";
+
+export default APIWrapper({
```

Comment:
Make sure that the nonprofit of the user owns the project before creating an issue.  Don't want nonprofits to be able to create issues for projects that are not theirs.  You can do it the same way I mentioned in the other PR.